nautobot_floor_plan/tables.py

```diff
@@ -43,3 +43,50 @@ class Meta(BaseTable.Meta):
             "tags",
             "actions",
         )
+
+
+class FloorPlanTileTable(BaseTable):
+    # pylint: disable=R0903
```

Comment:
Pylint supports rule disables by name as well as by code, which is more reader-friendly:

```suggestion
    # pylint: disable=too-few-public-methods
```